HousingRepairsSchedulingApi/Gateways/DrsAppointmentGateway.cs

```diff
@@ -67,16 +67,25 @@ public async Task<IEnumerable<AppointmentSlot>> GetAvailableAppointments(GetAvai
                 earliestDate = earliestDate.AddDays(_appointmentSearchTimeSpanInDays);
             }
 
+            // 60 day cutoff
+            var latestAppointmentDate = DateTime.Today.AddDays(60);
```

Comment:
DateTime.Today is "today at midnight" or in other words "The first millisecond of today"
Whereas I believe the intention is to include the 60th day in the appointments - so add 23 hours?

Reply:
Agreed that this is good enough to pass the requirements.